pygmt/src/blockm.py

```diff
@@ -132,13 +127,10 @@ def blockmean(table, outfile=None, **kwargs):
 
 
 @fmt_docstring
+@use_common(options=["V", "a", "f", "r"])
```

Comment:
My first impression is that this reduces some readability. What about using the full name as in `@use_common(options=["verbose", "aspatial", "coltypes", "registration"])`? Or do you think it's ok to use the short alias as is (I realize that it's easier to lookup the dictionary using `COMMON_OPTIONS[option]` where option is the short alias, compared to doing a reverse lookup).

Reply:
It would be helpful to get more opinions on this. You are correct that the current implementation is simpler but less readable. Either seems fine to me, but I do not have a great perspective on how problematic the readability could be for future contributors/maintainers who have more experience with PyGMT than GMT.